packages/shared/src/libs/helpers/get-relative-date/get-relative-date.helper.ts

```diff
@@ -0,0 +1,15 @@
+import { differenceInDays, formatDistance } from "date-fns";
+
+const getRelativeDate = (date: Date, baseDate: Date): string => {
+	const LESS_THAN_DAY = 0;
+
+	const daysDifference = differenceInDays(baseDate, date);
+
+	if (daysDifference === LESS_THAN_DAY) {
+		return "Updated today";
+	}
```

Comment:
1. we should always have Updated keyword, like `Updated 2 days ago`
2. this helper should only return relative date - today,  2 days ago etc. and we need to add `Updated ` where its used.
3. I think formatDistance should handle `today` somehow too, so that we don't need to calculate it manually
```suggestion
```

Reply:
or maybe there are other methods in date-fns, like formatRelative that will fit more

Reply:
already tried to find any, formatDistance seems the most appropriate, but not sure how to make it only date-based (without `15 hours ago`)

Reply:
We should be able to do this with formatRelative, checkout this example https://stackoverflow.com/a/49139748. In case this doesn't work, the second option is to remove time from today and use only date for comparison

Reply:
![image](https://github.com/user-attachments/assets/950c1aef-6a4c-4b78-94d6-b8e65ef2fd48)
Sorry need to clarify: is this option alright? Or are we trying to get `2 days ago`, `a month ago`, `a year ago`?

Reply:
looks ok to me

Reply:
thank a lot for helping :)